lib/formdata/0.0.1/index.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * FormData polyfill for k6
+ * Copyright (C) 2021 Load Impact
+ * License: MIT
+ *
+ * This simplifies the creation of multipart/form-data requests from k6 scripts.
+ * It was adapted from the original version by Rob Wu[1] to remove references of
+ * XMLHttpRequest and File related code which isn't supported in k6.
+ *
+ * [1]: https://gist.github.com/Rob--W/8b5adedd84c0d36aba64
+ **/
+(function(exports) {
+  if (exports.FormData) {
+    // Don't replace FormData if it already exists
+    return;
+  }
+  // Export variable to the global scope
+  exports.FormData = FormData;
```

Comment:
I am unsure why this closure is required? We [already have it in k6](https://github.com/loadimpact/k6/blob/46d53d99deba98add23079df93579408fa7ea094/js/initcontext.go#L203) anyway.

Reply:
Resolved by f7c6887.